lib/audited/auditor.rb

```diff
@@ -249,6 +249,8 @@ def audited_changes(for_touch: false)
 
         if for_touch
           filtered_changes.reject! do |k, v|
+            next unless audits.present?
+
             audits.last.audited_changes[k].to_json == v.to_json ||
             audits.last.audited_changes[k].to_json == v[1].to_json
```

Comment:
also it makes sense here to get `audits.last.audited_changes` once instead of twice per iteration.

Reply:
Great point, updated in https://github.com/collectiveidea/audited/pull/668